mobile/lib/features/wallet/scanner_screen.dart

```diff
@@ -37,38 +51,171 @@ class _ScannerScreenState extends State<ScannerScreen> {
 
   @override
   Widget build(BuildContext context) {
+    final walletService = context.read<WalletChangeNotifier>().service;
+
     return Scaffold(
-        appBar: AppBar(title: const Text("Scan")),
-        body: QRView(
-          key: qrKey,
-          onQRViewCreated: _onQRViewCreated,
-          overlay: QrScannerOverlayShape(
-              borderColor: Colors.red,
-              borderRadius: 10,
-              borderLength: 30,
-              borderWidth: 10,
-              cutOutSize: 300.0),
-          formatsAllowed: const [BarcodeFormat.qrcode],
-          onPermissionSet: (ctrl, p) => _onPermissionSet(context, ctrl, p),
-        ));
-  }
+        body: SafeArea(
+      child: Container(
+        margin: const EdgeInsets.only(top: 10),
+        child: Column(children: [
+          Container(
+            margin: const EdgeInsets.only(left: 10),
+            child: Row(
+              mainAxisAlignment: MainAxisAlignment.start,
+              children: [
+                GestureDetector(
+                  child: Container(
+                      alignment: AlignmentDirectional.topStart,
+                      decoration: BoxDecoration(
+                          color: Colors.transparent, borderRadius: BorderRadius.circular(10)),
+                      width: 70,
+                      child: const Icon(
+                        Icons.arrow_back_ios_new_rounded,
+                        size: 22,
+                      )),
+                  onTap: () {
+                    GoRouter.of(context).pop();
+                  },
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 20),
+          Expanded(
+              child: !cameraPermission
+                  ? const Center(
+                      child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
+                        Icon(FontAwesomeIcons.ban, size: 55),
+                        SizedBox(height: 20),
+                        Text("Grant 10101 camera permission to use the scanner!")
+                      ]),
```

Comment:
🔧 This message makes me feel a bit anxious haha. I like a more matter-of-fact sentence e.g. (based on Phoenix):

```suggestion
                      child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
                        Text("To scan a QR code you must give 10101 permission to access the camera.")
                      ]),
```

Reply:
Can we make it do a popup to direct the user to the settings for that, like many apps do?

Reply:
We already show a popup to request access. This screen is shown if the user denied access.

Reply:
Ah I see, lgtm in that case but agree with Lucas

Reply:
fyi: this is what the user sees the first time the scanner is opened.

![IMG_4069](https://github.com/get10101/10101/assets/382048/06175391-f76d-4a08-95df-6066e0f576f8)